lib/grape_fast_jsonapi/parser.rb

```diff
@@ -23,14 +23,15 @@ def call
           schema[:data][:example][:attributes][attribute] = send "#{type}_example"
         end
 
-        relationships_hash = model.relationships_to_serialize
+        relationships_hash = model.relationships_to_serialize || []
 
         relationships_hash.each do |model_type, relationship_data|
+          relationships_attributes = relationship_data.instance_values.symbolize_keys
```

Comment:
`#relationships_to_serialize` now returns a `FastJsonapi::Relationship` instance, which would cause a `NoMethodError (undefined method `[]' for #<FastJsonapi::Relationship:0x00007f97d26d1e30>):`,  ...probably changed since writing this


```
#<FastJsonapi::Relationship:0x00007f96fdb3b848
 @cached=nil,
 @conditional_proc=nil,
 @id_method_name=:owner_id,
 @key=:owner,
 @lazy_load_data=nil,
 @links={},
 @name=:owner,
 @object_block=nil,
 @object_method_name=:owner,
 @polymorphic=false,
 @record_type=:user,
 @relationship_type=:belongs_to,
 @serializer=:CompactUserSerializer,
 @transform_method=nil>
```